examples/fibre_channel/README.md

```diff
@@ -0,0 +1,87 @@
+<!-- BEGIN MUNGE: UNVERSIONED_WARNING -->
+
+<!-- BEGIN STRIP_FOR_RELEASE -->
+
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+
+<h2>PLEASE NOTE: This document applies to the HEAD of the source tree</h2>
+
+If you are using a released version of Kubernetes, you should
+refer to the docs that go with that version.
+
+<strong>
+The latest 1.0.x release of this document can be found
+[here](http://releases.k8s.io/release-1.0/examples/fibre_channel/README.md).
+
+Documentation for other releases can be found at
+[releases.k8s.io](http://releases.k8s.io).
+</strong>
+--
+
+<!-- END STRIP_FOR_RELEASE -->
+
+<!-- END MUNGE: UNVERSIONED_WARNING -->
+
+## Step 1. Setting up Fibre Channel Target
+
+On your FC SAN Zone manager, allocate and mask LUNs so Kubernetes hosts can access them.
+
+## Step 2. Creating the Pod with Fibre Channel persistent storage
+
+Once you have installed Fibre Channel initiator and new Kubernetes, you can create a pod based on my example [fc.yaml](fc.yaml). In the pod JSON, you need to provide *targetWWNs* (array of Fibre Channel target's World Wide Names), *lun*, and the type of the filesystem that has been created on the lun, and *readOnly* boolean.
+
+**Note:** If you have followed the instructions in the links above you
+may have partitioned the device, the Fibre Channel volume plugin does not
+currently support partitions so format the device as one partition.
+Make sure you have the correct device name then run the following as
+root to format it:
+
+```console
+mkfs.ext4 /dev/<name of device>
```

Comment:
@saad-ali I am thinking of a followup PR to do it for iscsi, rbd, and this fc. What's your take?